schemas/common/essa.schema.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
This file also appears in https://github.com/adobe/xdm/pull/824. Is that intended?

Reply:
Please ignore these files for this. I needed these files for my build. I will take the final files from Prabhakar's Oportunity schemas once its approved. Please ignore essa and essam for now. I already said that in my email